### ISSUE TITLE
[VT-126] Vit can't handle empty task list

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#147]
_Alejandro Lázaro on 2015-09-24T21:14:24Z says:_ 

 If i type vit:
Fatal error: default.command ("next") must print an "ID" column

If i type vit add hello:
Fatal error: default.command ("add hello") must print an "ID" column

But then, if i type vit again, it opens and shows a task named "hello". If i try to delete that task, it says that the task is deleted but also says ERROR:task next: no matches and the task entry still displays, but, if i close vit and type vit or task, it shows that the task was indeed deleted.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2015-09-24T21:14:24Z
Modified: 2015-09-25T01:39:05Z
```

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2015-09-25T01:37:42Z says:_ 

 Duplicate of VT-18.

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2015-09-25T01:39:05Z says:_ 

 I agree this needs to be fixed (maybe for VIT 1.4. 1.3 is almost done and I don't want to do any major restructuring for it at this point). Thanks for the feedback Alejandro.
